### PR TITLE
Add README note that ffmpeg v7 is incompatible with the project

### PR DESCRIPTION
**Pull request by andrew-pyle**
I built Shapez Community Edition from source today on MacOS M1 with ARM processor. I was able to successfully build the project.

I did have to resolve an error with FFmpeg, so I wanted to share the resolution.

```
[15:23:32] Plumber found unhandled error:
 Error: Output format mp3 is not available
...
[15:23:42] The following tasks did not complete: package.standalone-steam.darwin-arm64, <series>, 
  <series>, <parallel>, <series>, <series>, <parallel>, musicHQ, <series>, sfxOptimize
```

I stumbled on the solution on [StackOverflow](https://stackoverflow.com/questions/78451838/fluent-ffmpeg-error-with-mp3-output-format-when-streaming-audio-in-node-js). There are details there on whether/how the issue will be fixed upstream.

Basically, FFmpeg v7 is incompatible with the project's dependencies. This projects dependency `gulp-fluent-ffmpeg` itself depends on `fluent-ffmpeg`, which appears to be  [incompatible with FFmpeg v7](https://github.com/fluent-ffmpeg/node-fluent-ffmpeg/issues/1266).

**Solution: use FFmpeg v6 to build Shapez Community Edition.**

### CODE REVIEW

**Comment by totallytavi** [comment]
Author of SO answer. It's actually fixed now so you can just upgrade and it should be resolved
